Tangem/Modules/Markets/TokensNetworkSelector/MarketsTokensNetworkSelectorView.swift

```diff
@@ -14,83 +14,92 @@ struct MarketsTokensNetworkSelectorView: View {
     var body: some View {
         NavigationView {
             ZStack(alignment: .bottom) {
-                groupedContent
-            }
-            .alert(item: $viewModel.alert, content: { $0.alert })
-            .navigationBarTitle(Text(Localization.manageTokensNetworkSelectorTitle), displayMode: .inline)
-            .background(Colors.Background.tertiary.edgesIgnoringSafeArea(.all))
-        }
-    }
+                ScrollView {
+                    VStack(spacing: 14) {
+                        MarketsWalletSelectorView(viewModel: viewModel.walletSelectorViewModel)
 
-    private var groupedContent: some View {
-        GroupedScrollView {
-            MarketsWalletSelectorView(viewModel: viewModel.walletSelectorViewModel)
+                        contentView
 
-            if !viewModel.nativeSelectorItems.isEmpty {
-                Spacer(minLength: 14)
+                        MarketsGeneratedAddressView()
 
-                nativeNetworksContent
-
-                Spacer(minLength: 10)
-            }
-
-            if !viewModel.nonNativeSelectorItems.isEmpty {
-                Spacer(minLength: 14)
+                        Color.clear.frame(width: 10, height: 58, alignment: .center)
+                    }
+                    .padding(.horizontal, 16)
+                }
 
-                noneNativeNetworksContent
+                overlayButtonView
             }
+            .alert(item: $viewModel.alert, content: { $0.alert })
+            .navigationBarTitle(Text(Localization.manageTokensNetworkSelectorTitle), displayMode: .inline)
+            .background(Colors.Background.tertiary.edgesIgnoringSafeArea(.all))
         }
     }
 
-    private var nativeNetworksContent: some View {
-        VStack(alignment: .leading, spacing: 4) {
-            Text(Localization.manageTokensNetworkSelectorNativeTitle)
-                .style(Fonts.Bold.footnote, color: Colors.Text.tertiary)
-
-            Text(Localization.manageTokensNetworkSelectorNativeSubtitle)
-                .style(Fonts.Regular.caption1, color: Colors.Text.tertiary)
+    private var contentView: some View {
+        VStack(alignment: .leading, spacing: .zero) {
+            VStack(alignment: .leading, spacing: .zero) {
+                Text(Localization.marketsSelectNetwork)
+                    .style(Fonts.Bold.footnote, color: Colors.Text.tertiary)
 
-            Spacer(minLength: 10)
+                tokenInfoView
+            }
 
-            LazyVStack(spacing: 0) {
-                ForEach(viewModel.nativeSelectorItems) {
+            VStack(spacing: .zero) {
+                ForEach(viewModel.tokenItemViewModels) {
                     MarketsTokensNetworkSelectorItemView(viewModel: $0)
                 }
             }
-            .background(Colors.Background.action)
-            .cornerRadiusContinuous(Constants.cornerRadius)
+            .padding(.leading, 7)
```

Comment:
ошиблись в дизайне, надо им закинуть, чтобы поменяли на 8
```suggestion
            .padding(.leading, 8)
```